build/ci.yml

```diff
@@ -48,26 +49,42 @@ jobs:
   - template: steps-selfcontained.yml
   condition: ne(variables['Enable.Conda'], 'false')
 
-- job: "pack_conda_linux"
+- job: "pack_conda_macos"
   dependsOn: "pack_selfcontained"
   pool:
-    vmImage: 'ubuntu-latest'
+    vmImage: 'macOS-latest'
+  variables:
+    OS: 'mac'
+    CondaShellHook: "$('/usr/local/bin/conda' 'shell.bash' 'hook')"
   steps:
   - template: steps-conda.yml
   condition: ne(variables['Enable.Conda'], 'false')
 
-- job: "pack_conda_macos"
+- job: "pack_conda_linux"
   dependsOn: "pack_selfcontained"
   pool:
-    vmImage: 'macOS-latest'
+    vmImage: 'ubuntu-latest'
+  container:
+    image: qdkimages.azurecr.io/public/quantum/linux-selfcontained:latest
```

Comment:
For reliability, it would be good to move this over to `mcr.microsoft.com` (since `public`).